neural_compressor/config.py

```diff
@@ -1343,7 +1355,37 @@ def __init__(self):
         pass
 
 class Torch2ONNXConfig(ExportConfig):
-    """Config Class for Torch2ONNX."""
+    """Config Class for Torch2ONNX.
+
+    Args:
+        dtype (str, optional): The data type of the exported model, select from ["fp32", "int8"]. 
+                               Defaults to "int8".
+        opset_version (int, optional): The ONNX opset version used for export. Defaults to 14.
+        quant_format (str, optional): The quantization format of the exported int8 onnx model, 
+                                      select from ["QDQ", "QLinear"]. Defaults to "QDQ".
+        example_inputs (required): Example inputs used for tracing model. Defaults to None.
+        input_names (list, optional): A list of model input names. Defaults to None.
+        output_names (list, optional): A list of model output names. Defaults to None.
+        dynamic_axes (dict, optional): A dictionary of dynamic axes information. Defaults to None.
+        recipe (str, optional): A string to select recipes used for Linear -> Matmul + Add, select from 
+                                ["QDQ_OP_FP32_BIAS", "QDQ_OP_INT32_BIAS", "QDQ_OP_FP32_BIAS_QDQ"]. 
+                                Defaults to 'QDQ_OP_FP32_BIAS'.
+
```

Comment:
I think it is better if you can provide a link to different recipes so that users can understand what does they mean

Reply:
@chensuyue , Do we need to add a link to docs in docstring?

Reply:
I think we don't need a link in docstring.